src/server/pfs/fuse/mounter.go

```diff
@@ -2,13 +2,13 @@ package fuse
 
 import (
 	"fmt"
-	"log"
 	"os"
 	"os/signal"
 	"sync"
 
 	"bazil.org/fuse"
 	"bazil.org/fuse/fs"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Should we be using this library everywhere? Or is it just useful in this particular package for some reason?